src/telemetry/writer.hpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Author: Robertas Norkus
+ * Organisation: HYPED
+ * Date: January 2020
+ * Description:
+ *
+ *    Copyright 2019 HYPED
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#ifndef TELEMETRY_WRITER_HPP_
+#define TELEMETRY_WRITER_HPP_
+
+#include <rapidjson/writer.h>
+#include <string>
+#include "data/data.hpp"
+
+namespace hyped {
+namespace telemetry {
+
+  class Writer {
```

Comment:
Please document this class. Rapidjson is not well documented itself. It would be nice if you described how rapidjson works as well.